### PR TITLE
Add system time in console output

### PR DESCRIPTION
**Pull request by VVeremjova**
Add system time in console output for run test.
Add total executing time in log file and console.

Related to [APPLINK-27202](https://adc.luxoft.com/jira/browse/APPLINK-27202)

Please review @AGritsevich, @dtrunov, @pvvasilev, @OHerasym

### CODE REVIEW

**Comment by VVeremjova** [comment]
@AGritsevich 
After my changes console output will be look like this:
SDL pid 8363
**[17:27:02,966]** RunSDL************************************    SUCCESS (4 ms)
**[17:27:22,982]** InitHMI *************************************      SUCCESS (173 ms)
**[17:27:23,156]** InitHMI_onReady*************************                                SUCCESS (118 ms)
**[17:27:23,285]** ConnectMobile ***************************                    SUCCESS (3 ms)
**[17:27:23,290]** StartSession ******************************                             SUCCESS (2042 ms)
**Total executing time is 24448 ms**
Finish 'examples/Sample.lua'

**Comment by VVeremjova** [comment]
@AGritsevich See  	81dc4dc

File: modules/atf/util.lua [hunk 1]
```diff
@@ -63,12 +63,42 @@ function table.removeKey(t, k)
   return a
 end
 
+local function convertMs(milliseconds)
```
Comment (pestOO) [inline modules/atf/util.lua]: @VVeremjova 
Coudlw e use `qdatetime.get_datetime` indirectly?

Comment (pestOO) [inline modules/atf/util.lua]: @VVeremjova,
And could we re-write this to be more flexible?

**Comment by OHerasym** [comment]
Do we really need system time in each line?

**Comment by pvvasilev** [comment]
@VVeremjova Reviewed.

File: modules/atf/util.lua (hunk 1 above)
Comment (VVeremjova) [inline modules/atf/util.lua]: @pestOO I'm not sure, that format for Qdatetime will be appropriate for number, that will be given from lua side. If it isn't then all get_dattime should be rewritten and this convertion will be placed in  get_dattime. I don't think this is better then convertion in this file.

**Comment by VVeremjova** [comment]
@OHerasym See  	b458bea

File: modules/atf/util.lua (hunk 1 above)
Comment (pestOO) [inline modules/atf/util.lua]: @VVeremjova 
Can we use {{os.date}} and print time without msecs?

Comment (VVeremjova) [inline modules/atf/util.lua]: @pestOO No, test team needs msecs too

**Comment by pestOO** [comment]
@VVeremjova,
Please approbate using `QDateTime` from C++Core by upgrading `luaopen_qdatetime` or `timestamp`.
It could be more flexible and faster.

**Comment by VVeremjova** [comment]
@pestOO I have updated `luaopen_qdatetime` and found that Qt time format is not appropriate for timestamp. So should I rewrite all `convertMs` in QT or I can leave it?

**Pull request merged**